src/python_framework_v01/networkbuilder.py

```diff
@@ -273,11 +279,12 @@ def get_up_connections(
     downstream_key=r"downstream",
     verbose=False,
     debuglevel=0,
+    LOG=LOG,
```

Comment:
I understood that we don't need this explicit function argument here (i.e., ~~`LOG = LOG`~~)... will you check on that?

Reply:
That's correct. You don't need to pass the log around like that. You can either use a module level log or you can get a handle to the log via `getLogger`